fact_extractor/plugins/unpacking/dlink_shrs/code/dlink_shrs.py

```diff
@@ -1,24 +1,24 @@
 '''
 This plugin decrypts D-Link SHRS binaries.
 '''
+from subprocess import PIPE, STDOUT, run
 from pathlib import Path
+from shlex import split
 
-from common_helper_process import execute_shell_command
 
 NAME = 'D-Link SHRS'
 MIME_PATTERNS = ['firmware/dlink-shrs']
-VERSION = '0.1'
+VERSION = '0.1.1'
 
 TOOL_PATH = Path(__file__).parent.parent / 'internal/decrypt_dlink.py'
 
 
 def unpack_function(file_path, tmp_dir):
     decrypted_file = Path(tmp_dir, 'decrypted_image')
 
-    extraction_command = 'python3 {} -i {} -o {}'.format(TOOL_PATH, file_path, decrypted_file)
-    output = execute_shell_command(extraction_command)
-
-    return {'output': output}
+    extraction_command = f'python3 {TOOL_PATH} -i {file_path} -o {decrypted_file}'
```

Comment:
Why the changes to this file?

Reply:
There were problems with the tests and I added some debugging code which I then removed after it worked again. So the changes are no longer needed. I can restore the original version if you want.